src/entities/transactions/transaction.ts

```diff
@@ -85,9 +126,17 @@ export interface RainbowTransaction {
   explorerUrl?: string;
 }
 
+export enum FlashbotsStatus {
+  PENDING = 'PENDING',
+  INCLUDED = 'INCLUDED',
+  FAILED = 'FAILED',
+  CANCELLED = 'CANCELLED',
+  UNKNOWN = 'UNKNOWN',
+}
```

Comment:
these were used in a couple different files, so collocated it here and expanded it.